setup.py

```diff
@@ -9,7 +9,6 @@
 REQUIREMENTS = [
     "pandas>=0.23.1",
     "IndicoIo>=1.1.5",
-    "enso>=0.1.6",
```

Comment:
Is it necessary to have the requirements duplicated? Can we not reference the same requirements.txt in the setup.py?